packages/dotcom-build-sass/src/index.ts

```diff
@@ -23,6 +26,7 @@ export class PageKitSassPlugin {
       webpackImporter: this.webpackImporter,
       // Prefer `dart-sass`.
       implementation: require('sass'),
+      prependData: this.prependData,
```

Comment:
How do we feel about writing a test for this? I suppose since we're doing little and just passing through to webpack it may not be necessary?

Reply:
Going to see how easy it is to add a test for this.

Reply:
Looks like we could add something to the sass build example integration test.

Reply:
sorted